src/components/UserInfo.js

```diff
@@ -1,19 +1,23 @@
-// UserInfo.js
 export default class UserInfo {
-  constructor(nameSelector, jobSelector) {
-    this.nameElement = document.querySelector(nameSelector);
-    this.jobElement = document.querySelector(jobSelector);
+  constructor({ titleSelector, descriptionSelector, profileImage }) {
+    this._titleElement = document.querySelector(titleSelector);
+    this._descriptionElement = document.querySelector(descriptionSelector);
+    this._imageSelector = document.querySelector(profileImage);
```

Comment:
`_imageSelector`

Word `selector` means a **string** that is used for searching elements in the DOM